features/step_definitions/logging.rb

```diff
@@ -359,3 +359,22 @@
     step %Q/logging operators are installed successfully/
   }
 end
+
+Given /^logging eventrouter is installed in the cluster$/ do
+  step %Q/I switch to cluster admin pseudo user/
+  step %Q/I use the "openshift-logging" project/
+  unless deployment('eventrouter').exists?
+    @result = admin.cli_exec(:new_app, file: "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/eventrouter/internal_eventrouter.yaml")
+    unless @result[:success]
```

Comment:
Always do the clean-up not matter the above creation succeed or not, or we will left some resources there when met with partial success/failure.